slackbot/bot.py

```diff
@@ -19,7 +19,11 @@
 
 class Bot(object):
     def __init__(self):
-        self._client = SlackClient(settings.API_TOKEN)
+        self._client = SlackClient(
+            settings.API_TOKEN,
+            bot_icon = settings.BOT_ICON if hasattr(settings, 'BOT_ICON') else None,
```

Comment:
Please, follow PEP8 as much as possible. There should not be any space before/after the equal character of keyward arguments.